### ISSUE TITLE
Metrics

### ISSUE THREAD

**Issue by djanibekov** [iamsardorbek/Body-Performace-Data-Analysis#1]
Please try to use some other metrics, particularly for classification problems the "confusion matrix" is one of the most important metric

```
from sklearn.metrics import confusion_matrix
from sklearn.metrics import classification_report
```

**Comment by iamsardorbek** [comment]
Thank you for your comment. Just discovered instability in the metrics for my model, after implementing confusion matrix and classification report.

**Comment by iamsardorbek** [comment]
You can check the latest changes in the project.
@djanibekov
